test/unit/views/ArrivalSummaryViewSpec.scala

```diff
@@ -74,75 +74,91 @@ class ArrivalSummaryViewSpec extends ViewSpec with MockitoSugar with BeforeAndAf
   "View" should {
 
     "render title" in {
+
       page(answers).getTitle must containMessage("summary.arrival.title")
     }
 
     "render heading" in {
+
       page(answers).getElementById("title") must containMessage("summary.arrival.title")
     }
 
     "render 'Consignment details' section in summary list" in {
+
+      val answer_consignment_type_link_index = answer_consignment_type + 1
+      val answer_consignment_reference_link_index = answer_consignment_reference + 1
+
       val view = page(answers)
       view.getElementsByClass("govuk-heading-m").get(section_consignment_details) must containMessage("summary.consignmentDetails")
 
       view.getElementsByClass("govuk-summary-list__key").get(answer_consignment_type) must containMessage("summary.referenceType")
       view.getElementsByClass("govuk-summary-list__value").get(answer_consignment_type) must containMessage("consignmentReferences.reference.ducr")
 
-      val changeType = view.getElementsByClass("govuk-link").get(answer_consignment_type)
+      val changeType = view.getElementsByClass("govuk-link").get(answer_consignment_type_link_index)
```

Comment:
Ok dear.  I presume all of this "link" tests have been thrown out by the introduction of a new sign out link at the top of the page?

Reply:
Unfortunately yes :/